Integrations/integration-Cymon.yml

```diff
@@ -0,0 +1,481 @@
+
+        context[outputPaths['ip']] = {
+            'Address': ip,
+            'Malicious': {
+                'Vendor': 'Cymon',
+                'Description': description
+            }
+        }
+
+        context[outputPaths['ip']].update(asn_in_context);
+
+        return context
+
+    def get_ip_report_command():
+        args = demisto.args()
+
+        full_response = args.get('fullResponse') == 'true'
+
+        ip = args.get('ip')
+        if not is_ip_valid(ip):
+            return_error('An inalid IP was specified')
+
+        sources = get_ip_events_sources(ip)
+
+        if not sources:
+            return "IP " + ip + " is not in Cymons's dataset"
+
+        if (full_response):
+            max_len = 1000
+        else:
+            max_len = 50
+
+        events = get_ip_events(ip)
+        location = get_ip_location(ip)
+        domains = get_ip_domains(ip, max_len)
+        urls = get_ip_urls(ip, max_len)
+        asn = get_ip_asn(ip)
+
+        mark_down = create_ip_command_markdown(ip, sources, events, domains, urls, asn)
+        context = create_ip_command_context(ip, asn, events)
+
+        return [
+            {
+                'Type': entryTypes['map'],
+                'Contents': {
+                    'lat': float(location.get('lat')),
+                    'lng': float(location.get('lon'))
+                },
+                'ContentsFormat': formats['json']
+            },
+            {
+                'Type': entryTypes['note'],
+                'Contents': {
+                    'events': events,
+                    'sources': sources,
+                    'location': location,
+                    'domains': domains,
+                    'urls': urls,
+                    'asn': asn
+                },
+                'HumanReadable': mark_down,
+                'EntryContext': context,
+                'ContentsFormat': formats['json']
+            }]
+
+    ''' EXECUTION CODE '''
+    LOG('command is %s' % (demisto.command() ))
+    try:
+        command = demisto.command()
+        result = None
+
+        if command == 'test-module':
+            report = get_domain_report('google.com')
+
+            if 'total' in report:
+                result = 'ok'
+            else:
+                result = 'test failed'
+            pass
+        elif command == 'ip':
+            result = get_ip_report_command()
+        elif command == 'domain':
+            result = get_domain_report_command()
+
+        demisto.results(result)
+
+    except Exception, e:
+        LOG(e.message)
+        LOG.print_log()
+        raise
+  type: python
+  commands:
+    - name: ip
+      arguments:
+        - name: ip
+          required: true
+          default: true
+          description: Ip to check
+        - name: fullResponse
+          auto: PREDEFINED
+          predefined:
+            - "true"
+            - "false"
+          description: Return all of the results, note that it can be thousands of results.
```

Comment:
is this the case with Cymon? or just copy paste?

Reply:
fixed to "hundreds"